### DISCUSSION TITLE
Linebreak after EPD in SwissQRCode

### DISCUSSION THREAD

**Discussion by RamonaFrickCH** [Shane32/QRCoder#398] [General]
In Version 1.3.7
Wrote follow:
[Bugfix] Fixed missing linebreak when "Bill information" in SwissQRCode is empty

But on my QRCode information. I have always a linebreak. I use de Version 1.4.1 and I updated to 1.4.3 and in both version have a linebreak.
Why? What I doing wrong? I have unstructureMessage with "Steuern 2021" and BillInformation is Null. After EPD should not be a line break.

![RISC-5037](https://user-images.githubusercontent.com/105198572/167363429-65e4cbb8-ed1e-442c-ac26-579588e4728e.png)

![YYQRCODE RPX BOTTOM_ca6754be9f374c8697543f7ad32dd1e7](https://user-images.githubusercontent.com/105198572/167363530-810d8268-cf7f-45f1-bd9d-d45c01c13395.png)

**Comment by codebude** [comment]
This was a bug due to changes in the SwissQRCode standard 2.2. (The field StrdBkgInf switched from optional to additional.)
The bug was fixed by @RamonaFrickCH  in #411
